go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/tinygo-org/tinygo
 
-go 1.19
+go 1.21
```

Comment:
I think I saw somewhere on tinygo.org docs mentioning a requirement of Go 1.20 - would be good to confirm that and also bump the version in docs

Reply:
Not yet!  I think we still work with go 1.19.  So the proper workaround is probably a separate go.mod just for the spell checker for the moment.